src/_selectors/FullTradeSelectors.js

```diff
@@ -255,15 +255,24 @@ const availableAssetsFilter = (assets, times, now) => {
         if (!s.times) {
             return;
         }
-        const open = s.times.open[0];
-        const close = s.times.close[0];
+        const open = s.times.open;
+        const close = s.times.close;
 
         // Assuming closing time is larger than open time
         // TODO: some of Random does have time which does not fit into assumption
         if (s.name.indexOf('Random') > -1) {
```

Comment:
Will this work for a different language?
Also str.indexOf(...) > -1 can be str.includes(..)